changelog/fix-8386-remove-utc-label

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Transaction listing incorrectly labeled times as UTC
```

Comment:
Nitpick: Is it better to reword this changelog with a verb, or the action done?  eg: 
`Remove incorrect "UTC"  label from the time column of Transactions page`